internal/pkg/configloader/config.go

```diff
@@ -5,14 +5,14 @@ import (
 	"io/ioutil"
 	"runtime"
 
-	"github.com/merico-dev/stream/internal/pkg/log"
-
 	"gopkg.in/yaml.v3"
+
+	"github.com/merico-dev/stream/internal/pkg/log"
 )
 
 var (
-	GOOS   string = runtime.GOOS
-	GOARCH string = runtime.GOARCH
+	GOOS   = runtime.GOOS
```

Comment:
Please keep the original typed string constant, instead of using an untyped string constant. Unless there is a clear need for this.

Read more here: https://go.dev/blog/constants (section String constants)

Reply:
right